### PR TITLE
Adding branches strategy to README.md

### PR DESCRIPTION
**Pull request by AlexeySotkin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,3 +92,9 @@ To translate between LLVM IR and SPIR-V:
     * `-spirv-debug` - output debugging information
     * `-spirv-text` - read/write SPIR-V in an internal textual format for debugging purpose. The textual format is not defined by SPIR-V spec.
     * `-help` - to see full list of options
+
+## Branching strategy
+
+Code on the master branch in this repository is intended to be compatible with master/trunk branch of the [llvm](https://github.com/llvm-mirror/llvm) project. That is for an OpenCL kernel compiled to llvm bitcode by the latest version(bulit with latests commits ) of Clang it should be possible to translate it SPIR-V with llvm-spirv tool.
```
Comment (neildhickey) [inline README.md]: built is misspelled bulit. 
We need a "to" between "it" and "SPIR-V"

File: README.md [hunk 2]
```diff
@@ -92,3 +92,9 @@ To translate between LLVM IR and SPIR-V:
...
+All new development should be done on the master branch.
+To have versions compatible with release versions of LLVM, corresponding branches are created. For example, to build translator with LLVM 7.0 (`release_70` branch in the llvm repo) branch `llvm_release_70` should be used.
```
Comment (neildhickey) [inline README.md]: release->released

**Comment by neildhickey** [comment]
I've made a couple of comments in the file, not sure how to get them to display here

File: README.md (hunk 1 above)
Comment (neildhickey) [reply README.md]: latests should also be latest I think

File: .travis.yml [hunk 3]
```diff
@@ -13,6 +13,7 @@ git:
 branches:
   only:
     - master
+    - travis
```
Comment (pierremoreau) [inline .travis.yml]: Was this intended to be part of the pull request?

**Pull request merged**